tests/Cross-chain/Omnichain.ts

```diff
@@ -471,6 +471,9 @@ describe("Omnichain: ", async function () {
       "OmnichainGovernanceExecutor::cancel: proposal should be queued and not executed",
     );
   });
+  it("Return Pending state when proposal is not queued", async function () {
+    expect(await executor.state(1000)).to.equals(3);
```

Comment:
I think we should revert the call if the proposal doesn't exist, as we are doing in the GovernorBravo